resources_pages/install_ds_stack_ubuntu.md

```diff
@@ -233,8 +229,7 @@ To install packages individually, we can now use the following command: `conda i
 
 ```
 conda install \
- jupyterlab=2.* \
- numpy=1.* \
+ jupyterlab=3.* \
  pandas=1.* \
  flake8=3.* \
  black=19.*
```

Comment:
Not sure but shouldn't it be `black=21.*` like in the Mac instructions?

Reply:
Yup good catch